docs/streaming.md

```diff
@@ -0,0 +1,95 @@
+# Spark Streaming
+
+Spark Streaming is an extension of the core Spark API that enables scalable, high-throughput, fault-tolerant stream processing of live data streams. Data can be ingested from many sources like Kafka, Flume, Kinesis, or TCP sockets, and can be processed using complex algorithms expressed with high-level functions like map, reduce, join and window. 
+
+### The Basic Idea
+
+#### Spark Streaming
+Here is a basic Spark Streaming sample which writes to the console with `wordCounts.print()`:
```

Comment:
Let's change this example to show something other than word count so that it's different from the base example from the Spark Streaming docs. How about a simple filter (like throw away all lines that don't begin with "ERROR") or simple transformation (like uppercase the entire line). cc @korry8911